isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -2155,16 +2379,17 @@ namespace isobus
 		    (CAN_DATA_LENGTH == message->get_data_length()))
 		{
 			VirtualTerminalClient *parentVT = reinterpret_cast<VirtualTerminalClient *>(parentPointer);
+			auto &data = message->get_data();
```

Comment:
I've recently learned that `auto` doesn't deduce references, so this is actually copying the data. I suppose that means we should explicitly say `std::vector<std::uint8_t> &` rather than `auto &`